ecs-agent/api/ecs/client/ecs_client.go

```diff
@@ -21,6 +21,13 @@ import (
 	"strings"
 	"time"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
```

Comment:
nit(multiple occurrences): grouping

Reply:
This is following the group ordering of 1) stdlib, 2) thirdparty dependencies, 3) current project